bundles/org.openhab.voice.rustpotterks/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,94 @@
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="voice:rustpotterks">
+		<parameter-group name="wakewordDetector">
+			<label>Wakeword Detector</label>
+			<description>Wakeword detection options.</description>
+		</parameter-group>
+		<parameter-group name="noiseDetector">
+			<label>Noise Detector</label>
+			<description>Optional noise detection options.</description>
+		</parameter-group>
+		<parameter-group name="vadDetector">
+			<label>VAD Detector</label>
+			<description>Optional voice activity detector options.</description>
+		</parameter-group>
+		<parameter name="threshold" type="decimal" min="0" max="1" groupName="wakewordDetector">
+			<label>Threshold</label>
+			<description>Configures the detector threshold, is the min score (in range 0. to 1.) that some of the wakeword
+				templates should obtain to trigger a detection. Model defined value takes prevalence if present.</description>
+			<default>0.5</default>
+		</parameter>
+		<parameter name="averagedThreshold" type="decimal" min="0" max="1" groupName="wakewordDetector">
+			<label>Averaged Threshold</label>
+			<description>Configures the detector averaged threshold, is the min score (in range 0. to 1.) that the averaged
+				wakeword template should obtain to allow to continue with the detection. This way it can prevent to run the
+				comparison of the current frame against each of the wakeword templates. If set to 0 this functionality is disabled.</description>
+			<default>0.2</default>
+		</parameter>
+		<parameter name="comparatorRef" type="decimal" min="0" max="1" groupName="wakewordDetector">
+			<label>Comparator Ref</label>
+			<description>Configures the reference for the comparator used to match the samples.</description>
+			<default>0.22</default>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="comparatorBandSize" type="integer" groupName="wakewordDetector">
+			<label>Comparator Band Size</label>
+			<description>Configures the band-size for the comparator used to match the samples.</description>
+			<default>6</default>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="eagerMode" type="boolean" groupName="wakewordDetector">
+			<label>Eager Mode</label>
+			<description>Enables eager mode. End detection as soon as a result is over the score, instead of waiting to
+				see if the
+				next frame has a higher score.</description>
+			<default>true</default>
+		</parameter>
+		<parameter name="noiseDetectionMode" type="text" groupName="noiseDetector">
+			<label>Noise Detection Mode</label>
+			<description>Use a noise detector to reduce computation on absence of sound. Configures how difficult is to consider
+				a frame as noise (the required noise level).</description>
```

Comment:
Suggestion: Use a noise detector to reduce computation **in the** absence of sound. Configures **the difficulty** to consider a frame as noise (the required noise level).

To be also updated in README and i18n file.

Reply:
You fixed only one part. I believe "in the absence of sound" is more appropriate.

Reply:
> You fixed only one part. I believe "in the absence of sound" is more appropriate.

Missed that part, sorry.